<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPACalc</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>site_icon.png</normaloff>site_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>401</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QTableWidget" name="table">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Ders kodu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ders adı</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Harf notu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Kredi</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="button_clear_table">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Listeyi Temizle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_delete_course">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Seçili Olanları Sil</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_add_course">
             <property name="text">
              <string>Ders Ekle</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="button_pull">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>OBS'den Dersleri Çek</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_calculate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ortalama Hesapla</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>İTÜ Kepler'e Giriş Yap</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_add_course</sender>
   <signal>clicked()</signal>
   <receiver>button_add_course</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
